tf/start.sh

```diff
@@ -2,12 +2,13 @@
 
 set -e
 
-CONFIG=$1
+CONFIGDIR=$1
 GAMES=$2
 
-NET="/tmp/weights.txt"
-NETDIR="/home2/networks/upload"
+ROOT="/work/lc0"
+NETDIR="$ROOT/networks/upload"
 GAMEFILE="$HOME/.lc0.dat"
```

Comment:
If network of one size is trained, but pipeline crashed during training of larger net, restarting script would redo training of smaller network.

I don't see an elegant way to fix that, though, maybe in addition to global GAMEFILE have also per-network-size game network size, and check it in inner loop.

Reply:
I think this is worth fixing - alternative option: a persistent counter which is set to 0 at the end of all training and incremented in the training loop, then check it against a loop counter to decide how many loops can be skipped..

Reply:
That would break if we want to add/remove network size one day, or change number of games per cycle (very probable)

Reply:
It should be fine so long as we only modify the script by taking it down when its not training... ?

Reply:
@mooskagh that would make everything get out of sync wrt ramdisk data.

Maybe we should in that case remove the last checkpoint for nets that were already trained. At least stay consistent between networks.